.travis-docker.sh

```diff
@@ -86,9 +95,14 @@ fi
 
 case $opam_version in
     2)
+      opam_switch_selection=
+      if [ "$OCAML_BETA" = "enable" ]; then
+          echo "RUN opam repo add --dont-select beta $beta_repository" >> Dockerfile
+          opam_switch_selection="--repo=default,beta "
+      fi
       echo "RUN opam switch ${OCAML_VERSION} ||\
-          opam switch create ocaml-base-compiler.${OCAML_VERSION} ||\
-          opam switch create ${OCAML_VERSION}" >> Dockerfile ;;
+          opam switch create ${opam_switch_selection}ocaml-base-compiler.${OCAML_VERSION} ||\
+          opam switch create ${opam_switch_selection}${OCAML_VERSION}" >> Dockerfile ;;
```

Comment:
Can I suggest introducing `${ocaml_package}` which is `ocaml-base-compiler` unless `$OCAML_BETA` is `enable` when it becomes the required (for now) `ocaml-variants`?

Reply:
In fact, I think it might be more than suggest - doesn't this cause an error if `OCAML_VERSION` matches the system compiler version?

Reply:
There is no system compiler installed in the docker images and it should not be possible to install it anyway. Requiring `ocaml-variants` specifically seems ok since the third line was only added recently in https://github.com/ocaml/ocaml-ci-scripts/pull/318 and was a pretty much useless PR anyway.